gfx/portraits/portrait_modifiers/04_headgear_armor.txt

```diff
@@ -1083,4 +1083,51 @@
 			}
 		}
 	}
+
+	### ORCISH
+	orcish_war = {
+		dna_modifiers = {
+			accessory = {
+				mode = add
+				gene = headgear
+				template = orc_war_nobility_headgear
+				value = 0.6
+				range = { 0 1 } # For the randomness to work correctly
+			}
+		}
+		outfit_tags = { military_outfit }
+		weight = {
+			base = 0
+			modifier = {
```

Comment:
This modifier should probably be 

```
				portrait_war_nobles_helmet_trigger = {
					CULTURE_INPUT = orcish
				}
				OR = {
					portrait_commoner_armor_trigger = { 
						CULTURE_INPUT = orcish
					}
					portrait_low_nobles_armor_trigger = { 
						CULTURE_INPUT = orcish
					}
				}
```

Reply:
That way it will be consistent with all the other clothing in the game. If high nobles and royalty should use it to, add `portrait_high_nobles_armor_trigger` and/or `portrait_royalty_armor_trigger` in the same manner.